components/ContributorPopup.vue

```diff
@@ -53,15 +57,20 @@
         >
           <ul>
             <li
-              v-for="(number, key) of contributor.repositories"
-              :key="key"
-              class="contributions-item repository"
+              v-for="repository of contributor.repositories"
+              :key="repository.repositoryName"
             >
-              <p class="contribution-number">{{ number }}</p>
-
-              <p class="contribution-name">
-                {{ key.replace('PrestaShop/', '') }}
-              </p>
+              <a
+                :href="`https://github.com/${repository.repositoryName}`"
+                target="_blank"
+                class="contributions-item repository"
+              >
+                <p class="contribution-number">{{ repository.number }}</p>
+
+                <p class="contribution-name">
+                  {{ repository.repositoryName.replace('PrestaShop/', '') }}
```

Comment:
```suggestion
                  {{ repository.repositoryName.replace('PrestaShop/', '') }}
```

could be a computed :)